src/api/public/apidocs-new/paths/attribute_namespace_attribute_name.yaml

```diff
@@ -0,0 +1,30 @@
+delete:
+  summary: Delete an attribute and all its values in projects or packages.
```

Comment:
What are values in projects or packages?

Reply:
We talked about this offline, this will get more clear once we also document the `source_attribute` routes